docs/changelog/84391.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 84391
```

Comment:
@weizijun I don't think these changelog entries should be added for time series project? This feature is currently behind a feature flag and not ready yet to be used in production. The entire time series feature will likely have its own changelog entry when it is ready.

Reply:
done